build/assets/style.csl

```diff
@@ -10,7 +10,7 @@
       <name>Daniel Himmelstein</name>
       <email>daniel.himmelstein@gmail.com</email>
     </author>
-    <updated>2017-09-04T20:54:02+00:00</updated>
+    <updated>2018-08-22T19:08:46+00:00</updated>
     <rights license="https://creativecommons.org/publicdomain/zero/1.0/legalcode">This work is dedicated to the public domain via CC0 1.0</rights>
   </info>
   <macro name="author">
```

Comment:
As long as you are editing the style, is it easy to add a default value when there is no author?  I'll open a separate issue if it is out of scope.

See https://greenelab.github.io/meta-review/#ref-HPKoE9m3 for an example reference without an author.  The `title` and `venue` are concatenated without whitespace.